### DISCUSSION TITLE
Integration of different multiome library

### DISCUSSION THREAD

**Discussion by adrdufour** [GreenleafLab/ArchR#1459] [Questions / Documentation]
Hello,

I would like to know the best way to integrate multiple 10x multiome (scRNAseq + scATACseq) analyses together. I noticed that RNAseq have much more difficulties to be integrated together as you can see in the joined images.

Thanks for your responses.

![UMAP_J9](https://user-images.githubusercontent.com/16325905/172626767-09323013-2793-459a-b8c0-da68f724cc85.svg)

**Comment by rcorces** [comment]
I'm afraid I dont have specific suggestions for this. You can, of course, tweak the harmony parameters to try to get a better batch correction. But why not just use the LSI_ATAC for your dimensionality reduction and proceed with that? There is nothing that says you have to use a combined dim reduction or use any info on the RNA side for this.
